src/RTCRtpReceiver.ts

```diff
@@ -0,0 +1,36 @@
+import { NativeModules } from 'react-native';
+import MediaStreamTrack from './MediaStreamTrack';
+import RTCRtpCapabilities, { getCapabilities } from './RTCRtpCapabilities';
+
+const { WebRTCModule } = NativeModules;
+
+export default class RTCRtpReceiver {
+    _id: string;
+    _peerConnectionId: number;
+    _track: MediaStreamTrack;
+
+    constructor(info: { peerConnectionId: number, id: string, track: MediaStreamTrack }) {
+        this._id = info.id;
+        this._peerConnectionId = info.peerConnectionId;
+        this._track = info.track;
+    }
+
+    static getCapabilities(kind: "audio" | "video"): RTCRtpCapabilities {
+
+        if (kind == "audio") {
+            throw new Error("Unimplemented capabilities for audio");
```

Comment:
We could hardcode some, since we provide the matching WebRTC build...

Reply:
Could you elaborate more on that? the native side currently does not use AudioEncoderFactory so it wasn't possible to use the same hack for video.

Reply:
I mean that we could just hard code opus, isac, etc since the WebRTC build which this plugin uses will have those for sure.

Or maybe there is a way to query it, but I don't k on right off the bat.

Reply:
I couldn't find any method afaik. I'd say we hard code them for now.

Reply:
Done